applications/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -92,6 +92,20 @@ CONFIG_CONSOLE_GETCHAR=y
 # Main thread
 CONFIG_MAIN_THREAD_PRIORITY=7
 
+# AT CMD
+CONFIG_AT_CMD_RESPONSE_MAX_LEN=4096
```

Comment:
Out of curiosity - why do we need to extend this config even more?

Reply:
Removed in config cleanup